src/checks/stylelintConfigIsValid.ts

```diff
@@ -0,0 +1,104 @@
+import { promises as fs } from 'node:fs';
+import { createRequire } from 'node:module';
+import { execaCommand } from 'execa';
+import readdirp from 'readdirp';
+import semver from 'semver';
+import { projectPackageJson } from '../util/packageJson';
+import { supportedFileExtensions } from './stylelint';
+
+const projectDependencies = projectPackageJson.dependencies || {};
+const require = createRequire(`${process.cwd()}/`);
+
+export const title = 'Stylelint config is latest version';
+
+export default async function stylelintConfigIsValid() {
+  // Continue only if project is Next.js or UpLeveled React App
+  if (
+    !(
+      '@upleveled/react-scripts' in projectDependencies ||
+      'next' in projectDependencies
+    )
+  ) {
+    return;
+  }
```

Comment:
same comment as this: https://github.com/upleveled/preflight/pull/377/files/60fea13dd80aea4a8c37646614aadc5e16e2c05c#r1172916864

Reply:
https://github.com/upleveled/preflight/pull/377/commits/b93eaec0504d74657ca1ccc59430712f43a1e013